README.md

```diff
@@ -1,33 +1,89 @@
 # TyXML
 
-TyXML is a library to build statically correct Html5 and Svg documents.
+TyXML is an HTML and SVG templating library. It uses OCaml's type system to
+validate markup at compile time.
 
 ```ocaml
-let to_ocaml = Html5.(a ~a:[a_href "ocaml.org"] [pcdata "OCaml!"])
+open Tyxml
+
+let%html hello = "<li>Hello</li>"
+let%html world = "<li>World</li>"
+let%html image = "<img src='/like.png' alt='Like this'>"
+
+let%html greet = "<ul>" [hello; world; image] "</ul>"
+                                    (* ^^^^^
+> Error: This expression has type (... snip ...)
+         The second variant type does not allow tag(s) `Img *)
 ```
 
-TyXML provides a set of combinators to build Html5 and Svg documents. These combinators use the OCaml type-system to ensure the validity of the generated Html5 and Svg.
-TyXML's combinators are used in various libraries, such as [Eliom][] and [Js_of_ocaml][], but a builtin implementation is also provided by the `tyxml` ocamlfind package.
+Since `<ul>` elements should not contain `<img>` elements directly. If you
+remove `image` from the list in `greet`, the example compiles, and you can
+print:
 
-The documentation can be consulted [on the TyXML website](https://ocsigen.org/tyxml/manual/). Examples are available in the [examples](examples) directory.
+```ocaml
+Format.printf "%a" (Html.pp_elt ()) greet     (*
+> <ul><li>Hello</li><li>World!</li></ul> *)
+```
 
-[Eliom]: https://ocsigen.org/eliom/manual/clientserver-html
-[Js_of_ocaml]: https://ocsigen.org/js_of_ocaml/api/Tyxml_js
+TyXML checks element nesting, presence of correct attributes, attribute values,
+and some other properties.
+
+```ocaml
+let%html field = "<input type='pasword'>"
```

Comment:
[shame](https://www.youtube.com/watch?v=oJsOK9Tc60M) on me :)
"wrong_input_type" might be more explicit, but less likely as "real world mistake" example.
May be a commentary like
(* There is a mistake here: "pasword" was typed instead of "password". Hopefully the compiler will detect it*)